src/foam/core/reflow/ReactiveDetailView.js

```diff
@@ -386,11 +386,16 @@ foam.CLASS({
     ^ .foam-core-reflow-CollapsedByDefaultSectionView-actionDiv {
       display: flex;
       flex-direction: column;
+      gap: 10px;
     }
     ^ .foam-core-reflow-CollapsedByDefaultSectionView {
       border-bottom: 1px solid $borderLight;
       padding: 8px 16px;
     }
+    ^ .property-schedule .foam-u2-detail-VerticalDetailView {
+      border: 1px solid $borderLight;
+      border-radius: 4px;
+    }
```

Comment:
This change will fix the border in reflow but its broken everywhere else, can you fix it properly? If you change the dropdown to a select, you shouldnt need this anyway